resources/integration_test/tests/functions.yml

```diff
@@ -48,3 +48,28 @@ tests:
     expected_precision: [ 0, 10, 10, 0, 0, 0, 10, 10, 10, 10, 10, 10, 0, 10, 10, 10, 10, 10, 10 ]
     expected_scale: [ 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 ]
     expected_nullability: [ 0, 0, 1, 1, 1, 1, 0, 0, 0, 1, 0, 1, 1, 1, 1, 0, 1, 1, 1 ]
+  - description: SQLGetTypeInfo function getall types
+    test_definition: [ "sqlgettypeinfo", 0 ]
+    db: integration_test
+    expected_result:
+    - [ "string", 4294967287, null, "'", "'", null, 1, 1, 3, null, 0, null, "string", null, null, 4294967287, null, null, null ]
```

Comment:
That's odd, any idea why these `DATA_TYPE` values are in this format instead of a negative value?

Reply:
I _believe_ it is because [here](https://github.com/mongodb/mongo-odbc-driver/blob/7ac6e29497586f4af0e744cb10f06cd6fc45534c/integration_test/src/test_runner/mod.rs#L725) we cast to i64 (as opposed to i32), changing that line seems to yield the negative value. There are some existing tests structured this way too that I had followed, but I can update this + existing tests if you think that cast should be changed.

Reply:
I see.  Some of the existing tests would then benefit from it being cast to i32 as well to have the expected negative value instead of the two's complement.  Yes, let's update it to casting to i32.